### ISSUE TITLE
TransE bad training result

### ISSUE THREAD

**Issue by htang6** [yzhangee/NSCaching#2]
Thanks for your excellent work, but when I switch the model to transE while fixing other parameters unchanged, the top@10 retrieval rate doesn't improve during training and is still below to 1% after training for 300 epochs. The result looks good for ComplEx. Here is my command for training:

git clone https://github.com/yzhangee/NSCaching
cd NSCaching
git clone https://github.com/thunlp/OpenKE
mv OpenKE/benchmarks ../KG_Data
python train.py --model TransE

**Comment by yzhangee** [comment]
The hyper-parameters suitable for ComplEx may not be suitable for TransE. You can try to adjust the hyper-parameters, such as learning rate, weight decay, loss function, etc., first.

**Comment by htang6** [comment]
Thanks, I will try
